src/Enum/ToggleType.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5\Enum;
+
+/**
+ * Is an enumeration of toggle types.
+ */
+enum ToggleType: string
```

Comment:
Suggest group classes by widgets (`/Toggle/Toggle`, `/Toggle/ToggleType`), not by type (enum)

Reply:
Sounds logical, we can do it at the end of the whole refactoring.